docs/manual/themes/assets.rst

```diff
@@ -0,0 +1,87 @@
+Work with theme assets
+----------------------
+To install theme assets you need to run ``sylius:theme:assets:install`` command.
+
+Theme assets (JavaScript, CSS etc. files) should be placed inside the theme directory. There are few ways of reading
+theme assets in your Twig templates. The below how-to describes where to place the assets, how to install it and use it.
+
+Load assets from the theme's public directory (``app/themes/<theme-name>/public``)
```

Comment:
I would remove `(``app/themes/<theme-name>/public``)` from headline as it will render ugly in ToC.

Reply:
it was like that before, i just moved it to other file. and it looks ok in ToC
![zrzut ekranu 2018-01-17 o 15 30 29](https://user-images.githubusercontent.com/298610/35047820-6eaf32e8-fb9b-11e7-8839-d8066a2f3dfc.png)